app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+
+from typing import List
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, format_: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (format) VALUES (?)",
+            (format_,)
+        )
+        self._connection.commit()
+
+    def all(self) -> List[Actor]:
+        actor_managers_cursor = self._connection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"

```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.